build.py

```diff
@@ -44,6 +45,7 @@
 parser.add_option('-r', '--revision', dest='revision', action="store", help='Checkout a specific revision of the target', default=False)
 parser.add_option('-d', '--dev', dest='dev', action="store_true", help='enable developer mode (does not use target-locked.json)', default=False)
 parser.add_option('-g', '--generate-docs', dest='generate_docs', action="store_true", help='generate documentation for the current target', default=False)
+parser.add_option('-D', '--deploy', dest='deploy', action="store_true", help="Attempt to automatically flash the micro:bit.", default=False)
```

Comment:
Since -d is used for dev maybe we should change this to -f and --flash?

Reply:
Good shout.

Reply:
8bc82c2 does this.